corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html

```diff
@@ -56,6 +58,8 @@
     </div>
     {% endif %}
 
+    {{ request|toggle_tag_info:"CASE_LIST_TILE" }}
+    {% if request|toggle_enabled:'CASE_LIST_TILE' %}
```

Comment:
the `CASE_LIST_TILE` toggle allows user to set `useCaseTiles` property. If someone sets that to 'yes' and then the toggle is removed for the domain, the container below is still visible since `useCaseTiles` was set to `yes`.

Oh! i see. this now makes it unavailable in case just `CASE_LIST_CUSTOM_XML` toggle is enabled. Would need to add check for that toggle above instead of `data-bind` attr.